pos_debranding/static/src/xml/Chrome.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<templates id="template" xml:space="preserve">
+
+    <t
+        t-name="Chrome"
+        t-inherit="point_of_sale.Chrome"
+        t-inherit-mode="extension"
+        owl="1"
+    >
+        <xpath expr="//img[hasclass('pos-logo')]" position="replace" />
```

Comment:
I tend to prefer adding a `d-none` class to avoid breaking inheritance from any other module that could hook into that element.

Reply:
d-none don´t work well then i use: 

```
<attribute name="style">display: none;</attribute>
```

Reply:
Looks lile `oe_hidden` is the one to go in PoS (https://github.com/odoo/odoo/blob/14.0/addons/point_of_sale/static/src/css/pos.css#L57) So it looks like BS4 is not available in the POS